downloads/swan-lake-release-notes/swan-lake-beta2/RELEASE_NOTE.md

```diff
@@ -1,42 +1,217 @@
 
-#### For existing users
+## Updating Ballerina
 
-If you are already using jBallerina version 1.2.0, or above, you can directly update your distribution to jBallerina {{ site.data.stable-latest.metadata.version }} by executing the following command:
+If you are already using Ballerina, use the [Ballerina Update Tool](/learn/tooling-guide/cli-tools/update-tool/) to directly update to Ballerina Swan Lake Beta2. 
 
-> $ ballerina dist update
+First, run the command below to update the update tool to its latest version. 
 
-However, you need to use the following commands instead of the above if you have installed:
+> `bal update`
 
-- jBallerina 1.2.0 but switched to a previous version: `$ ballerina dist pull jballerina-{{ site.data.stable-latest.metadata.version }}`
-- a jBallerina version below 1.1.0: install via the [installers](/downloads/)
+If your **Update Tool is below version 0.8.14**, use the `ballerina update` command.
 
-#### For new users
+Next, run the command below to update to Swan Lake Beta2.
 
-If you have not installed jBallerina, then download the [installers](/downloads/) to install.
+> `bal dist pull slbeta2`
 
-### Highlights
+## Installing Ballerina
 
-### What is new in Ballerina Swan Lake Preview <VERSION>
+If you have not installed Ballerina, then download the [installers](/downloads/#swanlake) to install.
 
-#### Language
--
+## Language Updates
 
-#### Runtime
--
+### Improvements
 
-#### Standard Library
-- 
+- Introduced the support for recursive tuple types.
 
-#### Developer Tools
-- 
+```ballerina
+type RecursiveType [int, RecursiveType[]];
 
+public function main() {
+    RecursiveType a = [1];
+    RecursiveType b = [1, []];
+    RecursiveType c = [1, [a]];
+}
+```
+
+- Changed the static type of the string iteration from `string` to `string:Char`.
+
+```ballerina
+public function main() {
+    string str = "foo";
+
+    foreach string:Char s in str {
+        io:println(s);        
+    }
+
+    record {| string:Char value; |}? next = str.iterator().next();
+
+    if !(next is ()) {
+        io:println(next.value);
+    }
+}
+```
+
+### Bug Fixes
+
+To view bug fixes, see the [GitHub milestone for Swan Lake Beta2](https://github.com/ballerina-platform/ballerina-lang/issues?q=is%3Aissue+is%3Aclosed+milestone%3A%22Ballerina+Swan+Lake+-+Beta2%22+label%3AType%2FBug+label%3ATeam%2FCompilerFE).
+
+## Runtime Updates
+
+### Improvements
+
+- Introduced `configurable` variables of union types via the TOML syntax. 
+
+```ballerina
+configurable map<anydata> myMap = ?;
+configurable int|string id = ?;
+```
+
+**TOML:**
+
+```toml
+id = "12345"
+
+[myMap]
+name = "John"
+age = 10
+```
+
+### Bug Fixes
+
+To view bug fixes, see the [GitHub milestone for Swan Lake Beta2](https://github.com/ballerina-platform/ballerina-lang/issues?q=is%3Aissue+is%3Aclosed+milestone%3A%22Ballerina+Swan+Lake+-+Beta2%22+label%3AType%2FBug+label%3ATeam%2FjBallerina).
+
+## Standard Library Updates
+
+### New Features
+
+#### `postgresql` Package
```

Comment:
@daneshk Is this part of standard library  packages? This is ballerinax and we don't ship with distribution too right?

Reply:
yes, let's remove it from the release notes

Reply:
Removed. Thanks!